golioth/pytest_plugin.py

```diff
@@ -37,6 +41,14 @@ async def project(api_key):
 
 @pytest.fixture(scope="module")
 async def device(project, device_name):
-    device = await project.device_by_name(device_name)
+    if (device_name is not None):
```

Comment:
```suggestion
    if device_name is not None:
```
Those brackets are superfluous.